.env

```
@@ -188,3 +188,4 @@ VITE_FEATURE_THORCHAIN_TCY=true
 VITE_FEATURE_THORCHAIN_TCY_WIDGET=true
 
 VITE_FEATURE_TX_HISTORY_BYE_BYE=false
+VITE_FEATURE_RFOX_FOX_ECOSYSTEM_PAGE=false
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The VITE_FEATURE_RFOX_FOX_ECOSYSTEM_PAGE key should go before the VITE_FEATURE_TX_HISTORY_BYE_BYE key

(UnorderedKey)